exercises/exercise_1/README.md

```diff
@@ -1,336 +1,193 @@
-```sh
-$ docker run hashicorp/http-echo:latest --help
-Usage of /http-echo:
-  -listen string
-        address and port to listen (default ":5678")
-  -text string
-        text to put on the webpage
-  -version
-        display version information
-```
-**NOTE:** You can edit a file after touching it, using the PWD online editor.
+> **Time**: Approximately 10 minutes
+>
+> **Difficulty**: Easy
 
-![editor button](editor-button.png)
+## Table of Contents
 
-Touch your file before editing it, then click the refresh button:
-```sh
-$ touch docker-compose.yml
-```
-![editor](editor.png)
-
-You should have a file like the following:
-```yaml
-version: '3.7'
-services:
-  hello:
-    image: hashicorp/http-echo:latest
-    command: ["-text", "Hello DockerCon", "-listen",":8080"]
-    ports:
-     - 8080:8080
-```
+1. [Docker Compose Background](#docker-compose-background)
+2. [The Application](#the-application)
+3. [Finishing the Compose File](#finishing-the-compose-file)
+4. [Starting the Application Stack](#starting-the-application-stack)
+5. [Cleaning Up](#cleaning-up)
 
-## Deploy your Compose file
+## Exercise Objects
 
-Now that we have a compose file, we need to deploy it. There are two ways to do this:
-1. Using `docker-compose up`
-1. Using `docker stack deploy`
+By the end of this exercise, you will have:
 
-### Deploy using docker-compose
+- Learned about Docker Compose
+- Explored the Voting App application stack
 
-`docker-compose up` will use the `docker-compose.yml` file in the current directory.
+During this exercise we will build a Docker Compose file that is ready to deploy as a Docker Application.
 
-* **Try it now**:
+## Docker Compose Background
 
-```sh
-/workshop $ docker-compose up
-WARNING: The Docker Engine you're using is running in swarm mode.Compose does not use swarm mode to deploy services to multiple nodes in a swarm. All containers will be scheduled on the current node.
-To deploy your application across the swarm, use `docker stack deploy`.
-Creating network "workshop_default" with the default driver
-Pulling hello (hashicorp/http-echo:latest)...
-latest: Pulling from hashicorp/http-echo
-86399148984b: Pull complete
-Creating workshop_hello_1_b8ad94bc929a ... done
-Attaching to workshop_hello_1_550ca1a27072
-hello_1_550ca1a27072 | 2018/11/29 10:20:00 Server is listening on :8080
-```
+Docker Compose is a tool designed to make it easy to define and share multi-container applications. While you can do everything in Docker Compose with Docker CLI commands, defining it in a YAML file makes it easy to share. Plus, the application stack is able to be version controlled.
 
-Once deployed, a new link appears with the exposed port `8080`:
-![exposed port](open-port.png)
+For this exercise, we will start with an existing compose file. But, it has missing pieces in it, which we will fill in to learn about the various components of a compose file. We will then build on this file for the next exercises.
```

Comment:
Should we talk about `compose v2` vs `compose v3`? Or at least just say we will use only v3 as this is the only format supported by Docker App ?

Reply:
That's a good question. I don't think we should get into it too much. But, if we do, simply saying it is required by Docker App should suffice. I can add something like "Note that, when creating your own file in the future, only compose version 3 schema is supported in Docker App."